src/sentry/options/defaults.py

```diff
@@ -43,3 +43,5 @@
 # symbolizer specifics
 register('dsym.llvm-symbolizer-path', type=String)
 register('dsym.cache-path', type=String, default='/tmp/sentry-dsym-cache')
+
+register('mail.list-namespace', type=String, default='localhost', flags=FLAG_NOSTORE)
```

Comment:
I actually have no idea how `/manage/settings/` works now. If it's trivial to add, I'll just do it in this change.